debian/control

```diff
@@ -62,6 +62,7 @@ Depends: python3-apt,
          python3-xdg,
          python3-yaml,
          squashfs-tools,
+         openssh-sftp-server,
```

Comment:
That would effectively make project/cleanbuild deb-specific at runtime. Any particular reason to special-case sftp? Is there precedent for lazily installing packages in other cases?